appinventor/blocklyeditor/src/typeblock.js

```diff
@@ -159,6 +158,38 @@ Blockly.TypeBlock.prototype.handleKey = function(e){
       Blockly.mainWorkspace.hideChaff();
       return;
     }
+    switch (e.keyCode) {
+      case 9:   // Tab
+      case 16:  // Enter
+      case 17:  // Ctrl
+      case 18:  // Alt
+      case 19:  // Home
+      case 20:  // Caps Lock
+      case 33:  // PageUp
+      case 34:  // PageDown
+      case 35:  // Shift
+      case 36:  // End
+      case 45:  // Ins
+      case 91:  // Meta
+      case 112: // F1
+      case 113: // F2
+      case 114: // F3
+      case 115: // F4
+      case 116: // F5
+      case 117: // F6
+      case 118: // F7
+      case 119: // F8
+      case 120: // F9
+      case 121: // F10
+      case 122: // F11
+      case 123: // F12
+        return;
+    }
+    if (37 <= e.keyCode && e.keyCode <= 40) {
+      // TODO: try to add block.moveBy
```

Comment:
I'd like to not merge this TODO as it is right now. It may also affect the one's ability to use the arrow keys to navigate within the typeblocking editor.

Reply:
Done